inc/saiobject.h

```diff
@@ -102,6 +102,22 @@ typedef struct _sai_object_key_t
 
 } sai_object_key_t;
 
+typedef enum _sai_attr_stage_t
+{
+    /** Not applicable */
+    SAI_ATTR_STAGE_NA,
+
+    /** Common stage */
+    SAI_ATTR_STAGE_BOTH,
+
+    /** Ingress stage */
+    SAI_ATTR_STAGE_INGRESS,
+
+    /** Egress stage */
+    SAI_ATTR_STAGE_EGRESS
+
+} sai_attr_stage_t;
```

Comment:
I think sai_attr_stage_t is no longer used. Let us remove this and only use sai_object_stage_t.

Reply:
Done